src/broker/overlay.c

```diff
@@ -950,6 +965,11 @@ static void parent_cb (flux_reactor_t *r, flux_watcher_t *w,
 
     if (!(msg = zmqutil_msg_recv (ov->parent.zsock)))
         return;
+    /* Flag this message as remotely received. This allows efficient
+     * operation of the overlay_msg_is_remote() function.
+     */
+    if (flux_msg_aux_set (msg, "overlay::remote", int2ptr (1), NULL) < 0)
+        goto done;
```

Comment:
Call `logdrop()` on error?

Reply:
Oops, sorry, didn't look carefully enough to see that `logdrop()` is required/encouraged here.